### DISCUSSION TITLE
Just a quick thank you!

### DISCUSSION THREAD

**Discussion by brandonscollins** [thesamim/TickTickSync#117] [General]
Hello, I'm writing just to give some encouragement and say thank you for the hard work that has gone into this plugin. It provides a level of functionality that is not available in Obsidian anywhere else.

I looked for a way to send some money as a way of thanks but couldn't find anything. Let me know if that exists somewhere and I missed it.

Keep it up!

**Comment by johnthepleasant** [comment]
Same on my end! Thank you!

**Comment by thesamim** [comment]
I appreciate the encouragement folks! Thanks!
